docs/nodes/nodes-library/core-nodes/Webhook/README.md

```diff
@@ -123,6 +123,73 @@ Save the workflow and execute it again by clicking on the ***Execute Workflow***
 This example workflow uses the Webhook node, which is a Trigger node. You'll need to save the workflow and then click on the Activate toggle on the top right of the screen to activate the workflow. Your workflow will then be triggered every time a GET request is sent to the ***Production*** webhook URL.
 :::
 
+## FAQs
+
+### 1. Where to find the Webhook URLs?
+
+The Webhook node has two URLs - a Test URL and a Production URL.
+To get these URLs, follow the steps mentioned below.
+1. Click on ***Webhook URLs***. By default, the node displays the Production URL.
+2. If you want the Test URL, click on the ***Test*** tab.
+3. To copy the URL, click on the displayed URL.
+
+### 2. How to use cURL to trigger the Webhook node?
+
+You can use [cURL](https://curl.se/) to make HTTP requests that will trigger the Webhook node. To use cURL, make sure that you have installed it on your machine. You can follow [this guide](https://www.booleanworld.com/curl-command-tutorial-examples/) to install cURL on your machine.
+With cURL, you can make an HTTP request without any parameters. Moreover, you can make an HTTP request with a body parameter, header parameters, and you can even send files.
```

Comment:
This is a bit confusing. Perhaps take the first statement out and edit the second one accordingly?